src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -78,6 +78,21 @@ protected void SetUpEditor(string markupCode)
             }
         }
 
+        protected void SetEditorText(string contents)
```

Comment:
Or should Editor.SetText just do a clean setting of text without prettylisting and you can get rid of this helper completely?